src/main/java/todoly/fillforms/FillFormCreateAccount.java

```diff
@@ -0,0 +1,40 @@
+package todoly.fillforms;
+
+import core.selenium.utils.WebDriverHelper;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.Map;
+
+public class FillFormCreateAccount implements FillFormStrategy {
```

Comment:
Why is this class implementing FillFormStrategy if you are not using the methods?

Reply:
Do your tests run? Looks like Selenium Page Factory is not working